lib/radiator_web/controllers/api/task_controller.ex

```diff
@@ -0,0 +1,60 @@
+defmodule RadiatorWeb.Api.TaskController do
+  use RadiatorWeb, :rest_controller
+
+  alias Radiator.Task.TaskManager
+
+  alias Radiator.Directory.{
+    Editor,
+    Importer
+  }
+
+  def create(conn, %{"import_podcast_feed" => import_feed_params}) do
+    import_opts =
```

Comment:
I like to use Ecto for controller param whitelisting, validation and casting. Keeps the controller leaner / focused. I used to create a module for each but I think it's now possible inline with [schemaless changesets](https://hexdocs.pm/ecto/Ecto.Changeset.html?source=post_page---------------------------#module-schemaless-changesets).

But your way works fine as well.

Reply:
Interesting. Maybe worth a look.